packages/shared/src/components/post/write/CreatePostButton.tsx

```diff
@@ -74,11 +76,15 @@ export function CreatePostButton({
     >
       <Button
         {...buttonProps}
-        variant={sidebar ? ButtonVariant.Float : ButtonVariant.Secondary}
+        variant={
+          sidebar || footer ? ButtonVariant.Float : ButtonVariant.Secondary
```

Comment:
With the above check where we return null if !footer, wouldn't this always be true?

Reply:
The above check is not returning `null` if `!footer`, it just makes sure that the subsequent `(!user || !squads?.length)` check is ignored if `footer` is `true`, otherwise, the button is rendered if there is a `user` and there are postable `squads`.